velox/common/caching/SsdFile.cpp

```diff
@@ -144,6 +146,7 @@ SsdFile::SsdFile(const Config& config)
 #endif // linux
   fd_ = open(fileName_.c_str(), O_CREAT | O_RDWR | oDirect, S_IRUSR | S_IWUSR);
   if (fd_ < 0) {
+    RECORD_METRIC_VALUE(kMetricSsdCacheOpenSsdErrors)
```

Comment:
We shall also remove from Stats reporter? Thanks!

Reply:
Right they are in Velox now.. Somehow I thought they are still in Presto.

Reply:
s/kMetricSsdCacheOpenSsdErrors/kMetricSsdCacheOpenErrors/